biocrnpyler/integrase_enumerator.py

```diff
@@ -259,6 +260,13 @@ def reaction_allowed(self,site1,site2):
         if(tuple([site1.site_type,site2.site_type]) in self.reactions):
             return True
         return False
+    def reactive_sites(self):
```

Comment:
This can be simplified a bit:
```python
def reactive_sites(self):
    return list(set([list(reaction) for reaction in self.reactions]))

Reply:
i don't think so because i am starting with things like this `[(a,b),(c,d),(a,b)]` and I want to get `[a,b,c,d]`. Doing list comprehension on this nested tuple thing does not flatten the list.

Reply:
Oh, I see, I'll try to test out both version with the new tests you added to get a better understanding.